### PR TITLE
Update rules_ios to 2.2.1

### PR DESCRIPTION
**Pull request by luispadron**

### CODE REVIEW

File: examples/rules_ios/iOSApp/Test/ObjCUnitTests/BUILD [hunk 1]
```diff
@@ -15,7 +15,6 @@ rules_ios_ios_unit_test(
     name = "iOSAppObjCUnitTests",
     srcs = glob(["**/*.m"]),
     bundle_id = "rules-xcodeproj.example.objctests",
-    bundle_name = "iOS_App_ObjC_UnitTests",
```
Comment (luispadron) [inline examples/rules_ios/iOSApp/Test/ObjCUnitTests/BUILD:18]: Custom bundle_name was broken 😞  https://github.com/bazel-ios/rules_ios/pull/754#issuecomment-1731796129

Comment (thiagohmcruz) [reply examples/rules_ios/iOSApp/Test/ObjCUnitTests/BUILD:18]: I see you've got a [fix](https://github.com/bazel-ios/rules_ios/pull/770) up in `rules_ios` already, thx!

Comment (thiagohmcruz) [reply examples/rules_ios/iOSApp/Test/ObjCUnitTests/BUILD:18]: So once that lands we can bring this back correct?

**Review by thiagohmcruz — APPROVED** [review]

File: examples/rules_ios/iOSApp/Test/ObjCUnitTests/BUILD (hunk 1 above)
Comment (mattrobmattrob) [reply examples/rules_ios/iOSApp/Test/ObjCUnitTests/BUILD:18]: Mind commenting this out and linking to the rules_ios PR or waiting until a new version is published so this isn't lost?

**Review by mattrobmattrob — APPROVED** [review]

File: examples/rules_ios/iOSApp/Test/ObjCUnitTests/BUILD (hunk 1 above)
Comment (thiagohmcruz) [reply examples/rules_ios/iOSApp/Test/ObjCUnitTests/BUILD:18]: +1 to trying to pick the fixed code here instead of commenting it out if possible

Comment (luispadron) [reply examples/rules_ios/iOSApp/Test/ObjCUnitTests/BUILD]: Updated

**Pull request merged**